src/components/atoms/Select/Select.tsx

```diff
@@ -174,6 +173,9 @@ export const Select: React.FC<SelectProps> = ({
           </div>
         </div>
       </div>
+      <div className='icon-wrapper'>
+        {icon && <Icon icon={icon} />}
+      </div>
```

Comment:
maybe we could some sort of toggle for the position of the icon, because i think there might be cases, where we want it at the front

Reply:
The icon is still at the front here thanks to:

```css
  .icon-wrapper {
    display: flex;
    justify-content: center;
    align-items: center;
    position: absolute;
    bottom: 0;
    left: 0;
    ...
```